### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by akhilsagar2823**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.class
+Assignment2\
```
Comment (yashwantdhakad) [inline .gitignore:2]: you can merge your previous assignment into master or main branch and can create new branch for new assignment from master or main branch

File: Assignment1/UrlReader.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    public static void urlInput() throws IOException {
+        UrlReader writer = new UrlReader();
+        writer.input();
+    }
+
+    public void input() throws IOException {
+        while (true) {
+            Scanner sc = new Scanner(System.in);
+            BufferedWriter urlwriter = new BufferedWriter(
+                    new FileWriter("D:\\Internship\\P2 - Java Core\\Assignment\\Assignment1\\urls.txt", true));
```
Comment (yashwantdhakad) [inline Assignment1/UrlReader.java:22]: If you already place this url.txt file into the assignment then you can use the relative patch

**Review by yashwantdhakad — APPROVED** [review]
Approved with suggestions

File: .gitignore (hunk 1 above)
Comment (akhilsagar2823) [reply .gitignore:2]: > ok sir i will keep it in mind for next assignments

File: Assignment1/UrlReader.java (hunk 2 above)
Comment (akhilsagar2823) [reply Assignment1/UrlReader.java:22]: > If you already place this url.txt file into the assignment then you can use the relative patch

yes sir I'll correct it.

File: .gitignore (hunk 1 above)
Comment (akhilsagar2823) [reply .gitignore:2]: > you can merge your previous assignment into master or main branch and can create new branch for new assignment from master or main branch

yes sir I'll keep this in mind for further assignments.

**Pull request merged**